src/docker-outside-of-docker-compose/.devcontainer/library-scripts/docker-debian.sh

```diff
@@ -278,7 +278,7 @@ tee /usr/local/share/docker-init.sh > /dev/null \
 
```

Comment:
A bit separate from this PR, but do we still need this library-script approach, or can we adopt the devcontainer feature for this template?

Reply:
ℹ️  This Template has a dev config based upon `docker-compose` and we can not add Features object to `docker-compose.yaml`.

That said, I have been assuming that adding Features to `.devcontainer.json` might have unintended effects ? (eg. when user runs any docker compose commands) 🤔 I might be wrong but that's the only reason why I haven't switched library-scripts to Features for docker-compose scenarios.

Reply:
Features should work for docker-compose devcontainers. It adds the feature(s) to the primary container. Check out the test cases in the CLI repo for this scenario.

Reply:
Nice! I'll take a look and refactor docker-compose templates 👍